CorsaSchoolManagementSystem/studentDetails.php

```diff
@@ -6,6 +6,8 @@
     $users = new User(new Database());
     $template = new Template("Views/studentDetails.php");
     $studentDetails = $users->getStudentDetails(htmlspecialchars($_GET['studentId']));
```

Comment:
You can get the subjects from studentDetails, hence, no need to call the getClasses method. 
Check the getStudentDetails() query, you will notice the subjects have been fetched already